projects/packages/jetpack-mu-wpcom/bin/sync-newspack-blocks.sh

```diff
@@ -154,4 +156,36 @@ pnpm --package=jscodeshift dlx jscodeshift -t ./bin/sync-newspack-blocks-formatt
 echo "Changing PHP textdomain to match jetpack-mu-wpcom..."
 ../../../vendor/bin/phpcbf --standard=./.phpcs.dir.xml --filter=../../../vendor/automattic/jetpack-phpcs-filter/src/PhpcsFilter.php --runtime-set jetpack-filter-no-ignore -q $TARGET
 
+# Add textdomain to block.json
+echo "Adding textdomain to all block.json files..."
+for block_json_file in "$TARGET"/blocks/*/block.json; do
+	TMPFILE=$(mktemp)
+	jq --tab '. += {"textdomain": "jetpack-mu-wpcom"}' "$block_json_file" > "$TMPFILE"
+	mv "$TMPFILE" "$block_json_file"
+done
+
+# Generate PHPCS config file.
+echo "Generating .phpcs.dir.xml..."
+cat > "$TARGET"/.phpcs.dir.xml <<EOF
+<?xml version="1.0"?>
+<ruleset>
+
+	<rule ref="MediaWiki.Usage.ForbiddenFunctions">
+		<exclude name="MediaWiki.Usage.ForbiddenFunctions.isset"/>
+	</rule>
+
+</ruleset>
+EOF
+
+echo "Updating Phan baseline..."
+jetpack phan --update-baseline packages/jetpack-mu-wpcom
```

Comment:
This will be unneeded if we have phan ignore it entirely.

Reply:
877a79db16